### ISSUE TITLE
Add new-front/selfie appearance for video that forces use of the front-facing camera

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#1957]
There is currently a `selfie` appearance (soon to also have alias `new-front` #1848) for `images`. See https://github.com/opendatakit/collect/blob/d2bc5069bf8ab8f76506a829ac9a669c91a1d07b/collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieActivity.java for that implementation.

This is used in contexts where users are not very tech-savvy and need to take pictures of themselves for e.g. medical recording. A request for the same support for video has come in at https://forum.opendatakit.org/t/selfie-function-for-video/11743

**Comment by ArmenApresyan8** [comment]
I think we need to create a new activity which captures videos only by the front camera.

**Comment by Thodoris1999** [comment]
@opendatakit-bot claim

**Comment by Thodoris1999** [comment]
@lognaturel Should I do this with camera2 as well? I have finished the code with old camera api, but the new one is giving me a hard time... There is no documentation for recording videos and Google's example seems to be buggy on Lollipop https://github.com/googlesamples/android-Camera2Video/issues

**Comment by lognaturel** [comment]
> I have finished the code with old camera api

Awesome! Can you give us a sense of what the limitations are? It might be helpful to see a Work In Progress (WIP) PR to see whether what you have is sufficient.

**Comment by Thodoris1999** [comment]
I agree. I'll send my PR in a bit.

**Comment by lognaturel** [comment]
@opendatakit-bot you are so impatient! @Thodoris1999 is working on it! ;)

**Comment by Thodoris1999** [comment]
That's true! Just a little bit more and I'll finish it, I promise. However, I do owe you a progress report:

-The VerifyError was caused because android hates seeing imported packages it doesn't know about, so having a isFrontCameraAvailable() on CameraUtils was not possible. I could add a Camera2Utils, but even though camera2 video and image capture have some method similarities there are some differences that I'm scared to play with, so it wouldn't have much purpose besides that.

-I tried a lot of different approaches when configuring MediaRecorder and tampered with the preview, but none get rid of the problem on my device. Not finding many complete examples of veideo capture online doesn't help either. Maybe these logs mean something?
03-15 11:58:56.912 234-724/? E/Vold: Failed to find mounted volume for /storage/sdcard1/Android/data/org.odk.collect.android/files/
03-15 11:58:56.913 31547-31547/org.odk.collect.android W/ContextImpl: Failed to ensure directory: /storage/sdcard1/Android/data/org.odk.collect.android/files

I tried everything I could. My plan now is to cleanup my code and complete my pull request, so more tests on real devices can be done by reviewers and I get feedback from more experienced people
